Code/src/main/java/nl/utwente/group10/ui/handlers/ConnectionCreationManager.java

```diff
@@ -0,0 +1,91 @@
+package nl.utwente.group10.ui.handlers;
+
+import java.util.HashMap;
+import java.util.Map;
+
+
+import java.util.Optional;
+
+import javafx.geometry.Point2D;
+import javafx.scene.Parent;
+import nl.utwente.group10.ui.CustomUIPane;
+import nl.utwente.group10.ui.components.Connection;
+import nl.utwente.group10.ui.components.ConnectionAnchor;
+import nl.utwente.group10.ui.components.ConnectionLine;
+import nl.utwente.group10.ui.components.InputAnchor;
+import nl.utwente.group10.ui.components.OutputAnchor;
+
+public class ConnectionCreationManager {
```

Comment:
Naamgevingsdingetje: liever geen 'Manager'. Is het misschien een [Factory](https://en.wikipedia.org/wiki/Abstract_factory_pattern)? Of een [Builder](https://en.wikipedia.org/wiki/Builder_pattern)?